src/local-storage.store.ts

```diff
@@ -0,0 +1,31 @@
+import { AbstractStringValuedAsyncStore } from './abstract-string-valued.store';
+
+/*
+ * Removing stale keys is facilitating by storing the entire configuration into a single
+ * key in local storage. This is done by serializing the entire configuration object
+ * into a string and then storing it to a single key. When retrieving the configuration,
+ * we first check if the configuration exists in local storage. If it does, we deserialize
+ * the configuration from the string and return it. If the configuration does not exist,
+ * we return null.
+ */
+export class LocalStorageBackedAsyncStore<T> extends AbstractStringValuedAsyncStore<T> {
+  constructor(private localStorage: Storage, cooldownSeconds?: number) {
```

Comment:
added optoinal `cooldownSeconds?:` like we have for the chrome storage, that way the client has knowledge of the age of the data loaded from local storage, and can act appropriately.